### DISCUSSION TITLE
52.55.75.126(frontend .sh)error

### DISCUSSION THREAD

**Discussion by gorlelohitha** [learndevopsonline/learndevopsonline#1221] [Q&A] [answered]
Hi @r-devops  can you please help me with this?
while  i am executing  the script in frontend with sudo bash frontend.sh i am facing error like this

IMAGE:
![Screenshot (11)](https://github.com/learndevopsonline/learndevopsonline/assets/136321227/7e73f741-9fc4-47c6-8a7d-0ee6f281a899)

**Comment by maheshbusireddy** [comment, marked as answer]
@gorlelohitha you did a mistake in the nginx-roboshop.conf file. Actually your user server name is user.devopsb17.online by mistake you gave like  user.devops.17.online .i corrected that one now nginx service is working fine

![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/48fbbfd5-5332-4752-a201-a5edc89d7f54)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/ebc35832-c558-443d-8b6a-6908fb6d8a19)

**Reply by gorlelohitha** [reply]
Thankyou got it @r-devops
